Gruntfile.js

```diff
@@ -47,7 +42,7 @@ module.exports = function(grunt) {
             server: {
                 options: {
                     protocol: "https",
-                    hostname: "dashjs.local.vuplay.co.uk",
```

Comment:
Does the local https cert work with `localhost` as the DNS?

Reply:
yes indeed!